AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_CubeMapCaptureAdded.py

```diff
@@ -0,0 +1,116 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+class Tests:
+    cubemap_capture_creation = (
+        "CubeMap Capture Entity successfully created",
+        "P0: CubeMap Capture Entity failed to be created")
+    cubemap_capture_component = (
+        "Entity has a CubeMap Capture component",
+        "P0: Entity failed to find CubeMap Capture component")
+    cubemap_capture_component_removal = (
+        "UNDO CubeMap Capture Entity success",
+        "UNDO CubeMap Capture Entity failed")
+    removal_undo = (
+        "REDO CubeMap Capture Entity success",
+        "REDO CubeMap Capture Entity failed")
+    entity_deleted = (
+        "Entity deleted",
+        "P0: Entity was not deleted")
+    deletion_undo = (
+        "UNDO Entity deletion success",
+        "P0: UNDO Entity deletion failed")
+    deletion_redo = (
+        "REDO Entity deletion success",
+        "P0: REDO Entity deletion failed")
+
+
+def AtomEditorComponents_CubeMapCapture_AddedToEntity():
+    """
+    Summary:
+    Tests the Cubemap Capture component can be added to an entity and has the expected functionality.
+
+    Test setup:
+    - Wait for Editor idle loop.
+    - Open the "Base" level.
+
+    Expected Behavior:
+    The component can be added, and deleted.
+    Creation and deletion undo/redo should also work.
+
+    Test Steps:
+    1) Create a Cubemap_Capture entity with no components.
+    2) Add Cubemap_Capture component to Cubemap_Capture entity.
+    3) Remove the Cubemap_Capture component.
+    4) UNDO the Cubemap_Capture component removal.
+    5) Delete the Cubemap_Capture entity.
+    6) UNDO deletion.
+    7) REDO deletion.
+    8) Look for errors.
+
+    :return: None
+    """
+
+    import azlmbr.legacy.general as general
+
+    from editor_python_test_tools.editor_entity_utils import EditorEntity
+    import editor_python_test_tools.prefab_utils as PrefabUtils
```

Comment:
PrefabUtils its a class name style. this should be prefab_utils. I would order this so if it remains a straight import it comes before the other from import lines.

can you narrow the import since you are only using the one function.
from editor_python_test_tools.prefab_utils import wait_for_propagation
not certain that would work for you